string shape
bool scaled
int centreX
int centreY
int range
int side
int height
int weight
